<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="4" failed="1" total="6" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="67443" started-at="2016-05-27T09:11:35Z" finished-at="2016-05-27T09:12:42Z">
    <groups>
      <group name="UPDATE_CUSTOMER">
        <method signature="Update_Customer_Home_Address_001.rewardsLogin()[pri:0, instance:com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001@480bdb19]" name="rewardsLogin" class="com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001"/>
        <method signature="Update_Customer_Home_Address_001.updateExisitingCustomerDetails()[pri:0, instance:com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001@480bdb19]" name="updateExisitingCustomerDetails" class="com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001"/>
        <method signature="Update_Customer_Home_Address_001.rewardsSignOut_1()[pri:0, instance:com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001@480bdb19]" name="rewardsSignOut_1" class="com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001"/>
        <method signature="Update_Customer_Home_Address_001.rewardsSignin_2()[pri:0, instance:com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001@480bdb19]" name="rewardsSignin_2" class="com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001"/>
        <method signature="Update_Customer_Home_Address_001.staticValidationMyDetail()[pri:0, instance:com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001@480bdb19]" name="staticValidationMyDetail" class="com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001"/>
        <method signature="Update_Customer_Home_Address_001.rewardsSignOut_2()[pri:0, instance:com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001@480bdb19]" name="rewardsSignOut_2" class="com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001"/>
      </group> <!-- UPDATE_CUSTOMER -->
    </groups>
    <test name="Default test" duration-ms="67443" started-at="2016-05-27T09:11:35Z" finished-at="2016-05-27T09:12:42Z">
      <class name="com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001">
        <test-method status="PASS" signature="getBrowserName(java.lang.String)[pri:0, instance:com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001@480bdb19]" name="getBrowserName" is-config="true" duration-ms="16" started-at="2016-05-27T16:11:35Z" finished-at="2016-05-27T16:11:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getBrowserName -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001@480bdb19]" name="setUp" is-config="true" duration-ms="6131" started-at="2016-05-27T16:11:35Z" finished-at="2016-05-27T16:11:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="rewardsLogin()[pri:0, instance:com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001@480bdb19]" name="rewardsLogin" duration-ms="28418" started-at="2016-05-27T16:11:41Z" finished-at="2016-05-27T16:12:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- rewardsLogin -->
        <test-method status="FAIL" signature="updateExisitingCustomerDetails()[pri:0, instance:com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001@480bdb19]" name="updateExisitingCustomerDetails" duration-ms="31878" started-at="2016-05-27T16:12:09Z" depends-on-methods="com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001.rewardsLogin" finished-at="2016-05-27T16:12:41Z">
          <exception class="org.openqa.selenium.TimeoutException">
            <message>
              <![CDATA[Timed out after 10 seconds waiting for visibility of element located by By.xpath: //*[@id='my-detail-form']//span[@class='save-status-span' and contains(.,'Success')]
Build info: version: '2.53.0', revision: '35ae25b1534ae328c771e0856c93e187490ca824', time: '2016-03-15 10:43:46'
System info: host: 'Developer-205', ip: '10.59.0.92', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_77'
Driver info: org.openqa.selenium.phantomjs.PhantomJSDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, handlesAlerts=false, databaseEnabled=false, version=2.1.1, platform=XP, browserConnectionEnabled=false, proxy={proxyType=direct}, nativeEvents=true, acceptSslCerts=false, driverVersion=1.2.0, locationContextEnabled=false, webStorageEnabled=false, browserName=phantomjs, takesScreenshot=true, driverName=ghostdriver, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 049d4040-23eb-11e6-8441-058a3257d596]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.TimeoutException: Timed out after 10 seconds waiting for visibility of element located by By.xpath: //*[@id='my-detail-form']//span[@class='save-status-span' and contains(.,'Success')]
Build info: version: '2.53.0', revision: '35ae25b1534ae328c771e0856c93e187490ca824', time: '2016-03-15 10:43:46'
System info: host: 'Developer-205', ip: '10.59.0.92', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_77'
Driver info: org.openqa.selenium.phantomjs.PhantomJSDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, handlesAlerts=false, databaseEnabled=false, version=2.1.1, platform=XP, browserConnectionEnabled=false, proxy={proxyType=direct}, nativeEvents=true, acceptSslCerts=false, driverVersion=1.2.0, locationContextEnabled=false, webStorageEnabled=false, browserName=phantomjs, takesScreenshot=true, driverName=ghostdriver, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 049d4040-23eb-11e6-8441-058a3257d596
	at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:80)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:261)
	at com.automation.framework.util.Test.waitForElementVisible(Test.java:389)
	at com.automation.framework.util.Test.isExists(Test.java:223)
	at com.rabbitrewards.testscript.MyDetail.saveUpdateDetails(MyDetail.java:251)
	at com.rabbitrewards.testscript.MyDetail.updateDetailsForExistingCustomer(MyDetail.java:80)
	at com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001.updateExisitingCustomerDetails(Update_Customer_Home_Address_001.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
Caused by: org.openqa.selenium.NoSuchElementException: {"errorMessage":"Unable to find element with xpath '//*[@id='my-detail-form']//span[@class='save-status-span' and contains(.,'Success')]'","request":{"headers":{"Accept-Encoding":"gzip,deflate","Connection":"Keep-Alive","Content-Length":"112","Content-Type":"application/json; charset=utf-8","Host":"localhost:22260","User-Agent":"Apache-HttpClient/4.3.5 (java 1.5)"},"httpVersion":"1.1","method":"POST","post":"{\"using\":\"xpath\",\"value\":\"//*[@id='my-detail-form']//span[@class='save-status-span' and contains(.,'Success')]\"}","url":"/element","urlParsed":{"anchor":"","query":"","file":"element","directory":"/","path":"/element","relative":"/element","port":"","host":"","password":"","user":"","userInfo":"","authority":"","protocol":"","source":"/element","queryKey":{},"chunks":["element"]},"urlOriginal":"/session/049d4040-23eb-11e6-8441-058a3257d596/element"}}
Command duration or timeout: 10.48 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.53.0', revision: '35ae25b1534ae328c771e0856c93e187490ca824', time: '2016-03-15 10:43:46'
System info: host: 'Developer-205', ip: '10.59.0.92', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_77'
Driver info: org.openqa.selenium.phantomjs.PhantomJSDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, handlesAlerts=false, databaseEnabled=false, version=2.1.1, platform=XP, browserConnectionEnabled=false, proxy={proxyType=direct}, nativeEvents=true, acceptSslCerts=false, driverVersion=1.2.0, locationContextEnabled=false, webStorageEnabled=false, browserName=phantomjs, takesScreenshot=true, driverName=ghostdriver, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 049d4040-23eb-11e6-8441-058a3257d596
*** Element info: {Using=xpath, value=//*[@id='my-detail-form']//span[@class='save-status-span' and contains(.,'Success')]}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:500)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:355)
	at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:899)
	at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:41)
	at org.openqa.selenium.support.ui.ExpectedConditions$7.apply(ExpectedConditions.java:205)
	at org.openqa.selenium.support.ui.ExpectedConditions$7.apply(ExpectedConditions.java:201)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:238)
	... 29 more
Caused by: org.openqa.selenium.remote.ScreenshotException: Screen shot has been taken
Build info: version: '2.53.0', revision: '35ae25b1534ae328c771e0856c93e187490ca824', time: '2016-03-15 10:43:46'
System info: host: 'Developer-205', ip: '10.59.0.92', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_77'
Driver info: driver.version: RemoteWebDriver
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:138)
	... 39 more
Caused by: org.openqa.selenium.NoSuchElementException: {"errorMessage":"Unable to find element with xpath '//*[@id='my-detail-form']//span[@class='save-status-span' and contains(.,'Success')]'","request":{"headers":{"Accept-Encoding":"gzip,deflate","Connection":"Keep-Alive","Content-Length":"112","Content-Type":"application/json; charset=utf-8","Host":"localhost:22260","User-Agent":"Apache-HttpClient/4.3.5 (java 1.5)"},"httpVersion":"1.1","method":"POST","post":"{\"using\":\"xpath\",\"value\":\"//*[@id='my-detail-form']//span[@class='save-status-span' and contains(.,'Success')]\"}","url":"/element","urlParsed":{"anchor":"","query":"","file":"element","directory":"/","path":"/element","relative":"/element","port":"","host":"","password":"","user":"","userInfo":"","authority":"","protocol":"","source":"/element","queryKey":{},"chunks":["element"]},"urlOriginal":"/session/049d4040-23eb-11e6-8441-058a3257d596/element"}}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.53.0', revision: '35ae25b1534ae328c771e0856c93e187490ca824', time: '2016-03-15 10:43:46'
System info: host: 'Developer-205', ip: '10.59.0.92', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_77'
Driver info: driver.version: unknown
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.TimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- updateExisitingCustomerDetails -->
        <test-method status="SKIP" signature="rewardsSignOut_1()[pri:0, instance:com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001@480bdb19]" name="rewardsSignOut_1" duration-ms="0" started-at="2016-05-27T16:12:41Z" depends-on-methods="com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001.updateExisitingCustomerDetails" finished-at="2016-05-27T16:12:41Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method Update_Customer_Home_Address_001.rewardsSignOut_1()[pri:0, instance:com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001@480bdb19] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method Update_Customer_Home_Address_001.rewardsSignOut_1()[pri:0, instance:com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001@480bdb19] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- rewardsSignOut_1 -->
        <test-method status="SKIP" signature="rewardsSignin_2()[pri:0, instance:com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001@480bdb19]" name="rewardsSignin_2" duration-ms="1" started-at="2016-05-27T16:12:41Z" depends-on-methods="com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001.rewardsSignOut_1" finished-at="2016-05-27T16:12:41Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method Update_Customer_Home_Address_001.rewardsSignin_2()[pri:0, instance:com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001@480bdb19] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method Update_Customer_Home_Address_001.rewardsSignin_2()[pri:0, instance:com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001@480bdb19] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- rewardsSignin_2 -->
        <test-method status="SKIP" signature="staticValidationMyDetail()[pri:0, instance:com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001@480bdb19]" name="staticValidationMyDetail" duration-ms="0" started-at="2016-05-27T16:12:41Z" depends-on-methods="com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001.rewardsSignin_2" finished-at="2016-05-27T16:12:41Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method Update_Customer_Home_Address_001.staticValidationMyDetail()[pri:0, instance:com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001@480bdb19] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method Update_Customer_Home_Address_001.staticValidationMyDetail()[pri:0, instance:com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001@480bdb19] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- staticValidationMyDetail -->
        <test-method status="SKIP" signature="rewardsSignOut_2()[pri:0, instance:com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001@480bdb19]" name="rewardsSignOut_2" duration-ms="0" started-at="2016-05-27T16:12:41Z" depends-on-methods="com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001.staticValidationMyDetail" finished-at="2016-05-27T16:12:41Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method Update_Customer_Home_Address_001.rewardsSignOut_2()[pri:0, instance:com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001@480bdb19] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method Update_Customer_Home_Address_001.rewardsSignOut_2()[pri:0, instance:com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001@480bdb19] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1063)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- rewardsSignOut_2 -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001@480bdb19]" name="tearDown" is-config="true" duration-ms="956" started-at="2016-05-27T16:12:41Z" finished-at="2016-05-27T16:12:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.rabbitrewards.testcase.mydetail.Update_Customer_Home_Address_001 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
